### DISCUSSION TITLE
Github, git help

### DISCUSSION THREAD

**Discussion by Sas2k** [community/community#23832] [New to GitHub]
<p>hello i joined github this month and i am still getting use to it,<br>
Can anyone tell me how to use git with github<br>
as in transferring a git repo to a github repo etc…</p>

**Comment by Sas2k** [comment]
<p>also thanks in advance</p>

**Comment by jrocca82** [comment]
<p>Hi there!</p>
<p>Have you connected your computer to GitHub? You can connect Git to GitHub using SSH. Here is an article for more information:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/authentication/connecting-to-github-with-ssh">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/authentication/connecting-to-github-with-ssh" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/89630724/181079633-8e0ed724-eb0d-4449-a09c-ba092f638bfa.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/authentication/connecting-to-github-with-ssh" target="_blank" rel="noopener nofollow ugc">Connecting to GitHub with SSH - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>You only need to do this once, and then you can create, fork, clone, modify, and push repos from your computer onto your GitHub account.</p>
<p>Hope this helps!</p>

**Comment by Sas2k** [comment]
<p>thanks i will try that</p>

**Comment by Zerotask** [comment, marked as answer]
<p>Alternatively you can use HTTPS which is easier than SSH. You basically just have to authenticate with your GitHub account once (it will be stored in a local credentials store on your computer).<br>
You will find instructions on the already linked GitHub docs page above.</p>

**Comment by Sas2k** [comment]
<p>Ok I will try that method also thanks</p>
